### PR TITLE
backupccl: use correct option name in error message

### PR DESCRIPTION
**Pull request by stevendanna**
Previously, this error message referred to

    bulkio.backup.deprecated_full_backup_with_subdir.enable

but the actual option name is

    bulkio.backup.deprecated_full_backup_with_subdir.enabled

Release note (bug fix): An error message that referred to a
non-existent cluster setting no refers to the correct cluster setting:
bulkio.backup.deprecated_full_backup_with_subdir.enabled.

### CODE REVIEW

**Comment by stevendanna** [comment]
@msbutler The first commit is a small bug fix.  The second fix was a cleanup that made sense to me, but I'm fine dropping it if others don't agree.

**Comment by stevendanna** [comment]
Both of the remaining test failures appear to be flakes. Issues filed.

**Review by msbutler — APPROVED** [review]
dang, nice catch!

File: pkg/ccl/backupccl/backup_test.go [hunk 1]
```diff
@@ -6598,8 +6598,8 @@ INSERT INTO foo.bar VALUES (110), (210), (310), (410), (510)`)
 	resetStateVars()
 
 	tenant10.Exec(t, `
-CREATE DATABASE baz; 
-CREATE TABLE baz.bar(i int primary key, v string); 
+CREATE DATABASE baz;
```
[the remark this answers is not in the document]
Comment (stevendanna) [reply pkg/ccl/backupccl/backup_test.go:6601]: My editor cleans up extra whitespace at the end of the line. Typically I use `git add -p` and don't pull in such whitespace updates if they are in unrelated code but I guess I failed to do this. here. I'll fix this up before merging.

**Comment by stevendanna** [comment]
bors r=msbutler

**Pull request merged**